src/components/navigation/LoginCard.vue

```diff
@@ -1,52 +1,49 @@
 <template>
-  <v-card>
-    <v-card-title>Login</v-card-title>
-    <v-divider></v-divider>
-    <v-card-text>
-      <h4>Please select a realm to log in with</h4>
-    </v-card-text>
-    <v-card-actions>
-      <v-btn color="primary" @click="authEU" depressed>
-        EU
-      </v-btn>
-      <v-btn color="primary" @click="authNA" depressed>
-        NA
-      </v-btn>
-      <v-btn color="primary" @click="authASIA" depressed>
-        ASIA
-      </v-btn>
-      <v-btn color="primary" @click="authRU" depressed>
-        CIS
+  <v-card class="login-card">
+    <v-card-title class="login-card-title">
+      <div>{{ $t('navigation.login.card.title') }}</div>
+      <v-btn icon @click="closeHandler" v-if="isMobile">
+        <v-icon>mdi-close</v-icon>
```

Comment:
fa-times not mdi-close. We will be removing the mdi library from the project in the near future. It should never have been added